docs/deployment/feature_gates.md

```diff
@@ -39,6 +39,7 @@ The following tables are a summary of the feature gates that you can set on diff
 | ForceRestore                                 | `false` | `Alpha` | `1.39` |        |
 | DisableDNSProviderManagement                 | `false` | `Alpha` | `1.41` |        |
 | ShootCARotation                              | `false` | `Alpha` | `1.42` |        |
+| ShootSARotation                              | `false` | `Alpha` | `1.48` |        |
```

Comment:
> This PR does intentionally not include documentation since I would like to wait for https://github.com/gardener/gardener/pull/5939 to be merged. Respective documentation will be submitted accordingly afterwards.

As #5939 is now merged do you plan to add docs as part of this PR or will it be done in a separate PR?

Reply:
Yes, now added with 5e5a2bf.